src/manager/agent.manager.service.ts

```diff
@@ -240,8 +241,9 @@ export class AgentManagerService {
      * TODO check with the goverance policy on whether to allow the connection
      */
     public async connectAgent (agentId: string, adminApiKey: string): Promise<any> {
-        await this.pingConnectionWithRetry(agentId, process.env.AGENT_ADMIN_PORT, adminApiKey, parseInt(process.env.AGENT_RETRY_DURATION, 10));
-        const connectionData = await this.createConnection(agentId, process.env.AGENT_ADMIN_PORT, adminApiKey);
+        const agentCache: any = await this.cache.get(agentId);
```

Comment:
Can we make this so that if the cache value isn't found we use the default admin port?